### ISSUE TITLE
Use of LinterCop.json in pipeline

### ISSUE THREAD

**Issue by jwikman** [StefanMaron/BusinessCentral.LinterCop#263]
Hi Stefan,

I'm having issues with getting the LinterCop.json to work in my pipelines...

Where should the file be placed to work with Compile-AppInBcContainer?

It seems to be ignored if it's in the root folder (same folder as app.json) of the app being compiled. I've tried a few other places as well (as same folder as BusinessCentral.LinterCop.dll), but no luck...

**Comment by jwikman** [comment]
Ok, it seems as it works when i run alc.exe manually in the container if the LinterCop.json is in the same folder as alc.exe... 

But I just cannot get that to work in my pipeline (using `Compile-AppInBcContainer`), even if i put LinterCop.json in `C:\build\vsix\extension\bin` inside the container. 

@rvanbekkum, I know you are using LC in your pipelines - Are you also using LinterCop.json in the pipelines? If so, where do you put that file in the container?

**Comment by rvanbekkum** [comment]
> @rvanbekkum, I know you are using LC in your pipelines - Are you also using LinterCop.json in the pipelines? If so, where do you put that file in the container?

Hi @jwikman,
Yes, we are using it in our pipelines, but no, we do not have a `LinterCop.json`, so unfortunately I don't have a reference for you, sorry. 😅
My initial thoughts would be that it needs to be placed in the root directory of your app folder (where you also have your `app.json`), or that it needs to be in the agent/PowerShell work directory, but like I said, I have not tried it, so I am not sure.
Would be a good addition to the `README.md` if you find out :).

_Off-topic:_ What might also be a good addition to the `README.md` with regards to this is the information in this blog: https://stefanmaron.com/2022/01/11/business-central-lintercop-with-older-versions-of-bc/

**Comment by jwikman** [comment]
Thanks @rvanbekkum,

Too bad you didn't use that... 

I've tested
- in the app root (same as app.json)
- same folder as alc.exe (`C:\build\vsix\extension\bin`)
- same folder as `BusinessCentral.LinterCop.dll`, which I've tried in different places as well:
  - same folder as alc.exe (`C:\build\vsix\extension\bin`)
  - the Analyzer folder (`C:\build\vsix\extension\bin\Analyzer`)
  - Any other folder shared with the container
  - The PowerShell script folder

As seen at https://github.com/StefanMaron/BusinessCentral.LinterCop/blob/954ce9407f1865865f4feed1023283cc5ab1dc17/Helpers/LinterSettings.cs#L19, `LinterCop.json` is opened without path. And fromt the [docs](https://learn.microsoft.com/en-us/dotnet/api/system.io.file.opentext?view=net-7.0#remarks)  this means that it is opened from "the current working" directory.   And I probably "just" need to figure out which folder that is when running the pipeline. I'll see if I can add some debugging info in the pipeline.

I'll get back and update this issue and the `README.me` if I figure this out.
